### DISCUSSION TITLE
Problem With my WinDynamicDestop

### DISCUSSION THREAD

**Discussion by MikkeyDamn** [t1m0thyj/WinDynamicDesktop#466] [General]
It shows that my  WinDynamicDestop keeps showing this.
<img width="298" alt="Screenshot 2022-07-18 163135" src="https://user-images.githubusercontent.com/109058342/179473695-3697a548-706c-4460-afa8-023ffd5cf30d.png">
 
 Pls Help me with this

**Comment by t1m0thyj** [comment]
Have you used the option to [hide the tray icon](https://github.com/t1m0thyj/WinDynamicDesktop/wiki/Frequently-asked-questions#how-can-i-hide-the-tray-icon)? Or do you have multiple versions of WDD installed (both from GitHub and the Microsoft Store)?

Otherwise if don't see the WDD icon in your system tray, I'm not sure why there would be multiple instances of the app running. You could also check in Task Manager to see if there is already a WinDynamicDesktop app running in the background.

**Comment by mw10987** [comment]
I installed WinDynamicDesktop then included a bunch of themes and it was working fine but never worked after that. I tried re-installing it. Using the Microsoft Store version. The portable version. Same problem. WHat's wrong and what can I do?
![IMG_2500](https://user-images.githubusercontent.com/96133895/236649368-eeb65438-60ea-4525-8219-2d7b634da3a5.jpg)
![IMG_2501](https://user-images.githubusercontent.com/96133895/236649369-4b12e16d-99db-4f8c-9c9a-33e605e9161c.jpg)
[WinDynamicDesktop_5.2.1_x64_Portable.exe.log](https://github.com/t1m0thyj/WinDynamicDesktop/files/11413508/WinDynamicDesktop_5.2.1_x64_Portable.exe.log)

**Reply by t1m0thyj** [reply]
@mw10987 The logs show an Out of Memory error. How much RAM does your system have? And if you open Task Manager, how much RAM is WinDD using?
